projectroles/management/commands/cleanappsettings.py

```diff
@@ -42,10 +42,29 @@ def handle(self, *args, **options):
             else:
                 def_kwargs['app_name'] = 'projectroles'
             try:
-                app_settings.get_definition(**def_kwargs)
+                definition = app_settings.get_definition(**def_kwargs)
+                if s.project and s.project.type not in definition.get(
+                    'project_types', ['PROJECT']
+                ):
+                    logger.info(
+                        'Deleting "{}" from project "{}" - project type "{}" does not match allowed types: {}'.format(
```

Comment:
A minor point but the ` - ` syntax is not consistent with SODAR Core logging messages, I recommend using `: ` instead.